code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -224,10 +224,10 @@
 	desc = "A weapon for combat exosuits. Shoots a spread of pellets."
 	icon_state = "mecha_scatter"
 	equip_cooldown = 20
-	projectile = /obj/item/projectile/bullet/midbullet
-	projectiles = 40
+	projectile = /obj/item/projectile/bullet/pellet
+	projectiles = 90
 	projectile_energy_cost = 25
-	var/projectiles_per_shot = 4
+	var/projectiles_per_shot = 9	//81 points of damage if all pellets hit, seems fine to me for a mech shotgun.

```

Comment:
it was part of an attempted fix. The original projectiles all together did 80 damage anyway, so it'd literally be one more point of damage IF it even worked, which it doesn't because it's a bigger issue than that.